src/features/game/events/landExpansion/moveCollectible.ts

```diff
@@ -10,6 +10,7 @@ export const MOVE_COLLECTIBLE_SECONDS: Partial<
   "Basic Scarecrow": 60 * 10,
   "Emerald Turtle": 60 * 60 * 24 + 100,
   "Tin Turtle": 60 * 60 * 4 + 100,
+  Bale: 60 * 60 * 48 + 100,
```

Comment:
This is a large cool down - is there anything we can hint at when moving the Bale that it will be inactive for 48 hours?